x/payments/internal/types/util.go

```diff
@@ -73,9 +74,9 @@ func NewMsgEffectPayment(contractId string, creatorDid did.Did) MsgEffectPayment
 	}
 }
 
-func CheckNotEmpty(value string, name string) (valid bool, err sdk.Error) {
+func CheckNotEmpty(value string, name string) (valid bool, err error) {
 	if strings.TrimSpace(value) == "" {
-		return false, sdk.ErrUnknownRequest(name + " is empty.")
+		return false, sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, "name + \" is empty.\"")
```

Comment:
Please output the actual value of the `name` variable rather than "name" as a string. Consider using `Wrapf`

Reply:
added